devicetypes/smartthings/zigbee-motion-sensor-light.src/led-cpx-light.groovy

```diff
@@ -65,24 +65,29 @@ private getMOTION_CLUSTER() { 0x0406 }
 private getMOTION_STATUS_ATTRIBUTE() { 0x0000 }
 private getON_OFF_CLUSTER_VALUE() { 0x0006 }
 private getCONFIGURE_REPORTING_RESPONSE() { 0x07 }
+private getON_DATA_VALUE() { [0x01, 0x00] }
+private getOFF_DATA_VALUE() { [0x00, 0x00] }
```

Comment:
Can we loose that `_VALUE` end? Just this time make sure that you're editing _every_ occurrence.

Reply:
Sure, I changed those.